common/djangoapps/student/tests/test_create_account.py

```diff
@@ -32,6 +32,22 @@
 TEST_CS_URL = 'https://comments.service.test:123/'
 
 
+def path_decorator(patch_string):
+    def real_decorator(func):
+        def wrapper(func_obj, email, is_avaliable, expect_success, **kwargs):
+            mock_object = MagicMock()
+            attrs = {
+                'filter.return_value': mock_object,
+                'exclude.return_value': mock_object,
+                'exists.return_value': is_avaliable,
+            }
+            mock_object.configure_mock(**attrs)
+            with patch(patch_string, mock_object):
+                result = func(func_obj, email, expect_success)
```

Comment:
I would omit result declaration and just `return func(func_obj, email, expect_success)`